examples/marbles/frommarbles_merge.py

```diff
@@ -13,3 +15,5 @@
     on_completed=lambda: print('good job!'),
     scheduler=ccy.timeout_scheduler,
     )
+
```

Comment:
Have you considered using `.run()` instead? It's made for exactly this scenario. It will then return the last value emitted, and you can pipe to `ops.to_iterable()` to catch them all.